kie-maven-plugin/src/main/java/org/kie/maven/plugin/BuildMojo.java

```diff
@@ -25,38 +25,27 @@
 import org.apache.maven.plugins.annotations.Parameter;
 import org.apache.maven.plugins.annotations.ResolutionScope;
 import org.apache.maven.project.MavenProject;
-import org.drools.compiler.compiler.BPMN2ProcessFactory;
-import org.drools.compiler.compiler.DecisionTableFactory;
-import org.drools.compiler.compiler.GuidedDecisionTableFactory;
-import org.drools.compiler.compiler.GuidedRuleTemplateFactory;
-import org.drools.compiler.compiler.GuidedScoreCardFactory;
-import org.drools.compiler.compiler.PMMLCompilerFactory;
-import org.drools.compiler.compiler.ProcessBuilderFactory;
-import org.drools.compiler.kie.builder.impl.InternalKieModule;
-import org.drools.compiler.kie.builder.impl.KieContainerImpl;
-import org.drools.compiler.kie.builder.impl.KieMetaInfoBuilder;
-import org.drools.compiler.kie.builder.impl.KieProject;
-import org.drools.compiler.kie.builder.impl.ResultsImpl;
-import org.drools.compiler.kie.builder.impl.ZipKieModule;
+import org.codehaus.plexus.PlexusContainer;
+import org.codehaus.plexus.component.repository.exception.ComponentLookupException;
+import org.drools.compiler.compiler.*;
```

Comment:
Do you mean disabling the IDE import optimization to avoid the use of * instead to declare explicitely all imports ?